src/main/index.ts

```diff
@@ -32,6 +32,7 @@ import {
 } from '../types';
 import { checkHolochainLairBinariesExist } from './binaries';
 import { validateArgs } from './cli';
+import { searchHeigh, windowSize } from './const';
```

Comment:
This is a matter of taste but I like global constants to be all capital (e.g. `SEARCH_HEIGHT`) to make it clear that they are not dynamically assigned somewhere based on some logic but hard-coded.

Reply:
Let's keep all capital convention, I will change that